ISSUE_TEMPLATE.md

```diff
@@ -1,12 +1,14 @@
 ## My Environment
 
-* __ArangoDB Version__:        <!-- e.g. 3.3.14 or self-compiled devel branch -->
+* __ArangoDB Version__:        <!-- e.g. 3.3.19 or self-compiled devel branch -->
 * __Storage Engine__:             <!-- MMFiles / RocksDB -->
 * __Deployment Mode__:       <!-- Single Server | Master/Slave | Active Failover | Cluster | DC2DC -->
 * __Deployment Strategy__:   <!-- Manual Start | Manual Start in Docker | ArangoDB Starter | ArangoDB Starter in Docker | Kubernetes | DCOS -->
 * __Configuration__:               <!-- cluster setup details, notable server settings, etc. -->
 * __Infrastructure__:               <!-- AWS | Azure | ... | own -->
 * __Operating System__:        <!-- Ubuntu 18.04 | Windows 10 | MacOS 10.13.4 | DCOS 1.10 | ... -->
+* __Total RAM in your machine__:        <!-- e.g. 32Gb -->
+* __Disks in use__:        <!-- SSD | HDD -->
```

Comment:
What about multi-node setups? Could be multiple machines. Do we need to know per node/machine RAM or total? Same for disks, there could be differently sized storage media.

Reply:
for all, ideally - committed something